docs/user/integrations/dna_center.md

```diff
@@ -0,0 +1,81 @@
+# Cisco DNA Center SSoT Integration
+
+The Cisco DNA Center SSoT integration is built as part of the [Nautobot Single Source of Truth (SSoT)](https://github.com/nautobot/nautobot-app-ssot) app. The SSoT app enables Nautobot to be the aggregation point for data coming from multiple systems of record (SoR).
+
+From Cisco DNA Center into Nautobot, it synchronizes the following objects:
+
+| DNA Center              | Nautobot                     |
+| ----------------------- | ---------------------------- |
+| Areas                   | Location*                    |
+| Buildings               | Location*                    |
+| Floors                  | Location*                    |
+| Devices                 | Devices                      |
+| Ports                   | Interfaces                   |
+| Prefixes                | Prefixes                     |
+| IP Addresses            | IP Addresses                 |
+
+`*` As of SSoT 3.2.0 the LocationType for Areas, Buildings, and Floors can be defined in the Job form. Prior to SSoT 3.2.0 the DNA Center integration creates a Region, Site, and Floor LocationType and imports Areas as Region Locations, Buildings as Site Locations, and Floors as Floor Locations.
+
+`**` If the [Device Lifecycle Nautobot app](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt) is found to be installed, a matching Version will be created with a RelationshipAssociation connecting the Device and that Version.
```

Comment:
Does the `**` need to be next to `Devices` in the table above?

Reply:
Good point. I've added it.